### ISSUE TITLE
Issue: Javascript Overlay Blocks PDF Viewer in Chrome Version 66.0.3359.181

### ISSUE THREAD

**Issue by nirtiac** [gkiar/gitmarklet#56]
When the bookmark is selected while viewing a PDF file in the default PDF viewer, the PDF viewer turns black and the PDF is no longer displayed. This does not occur in Safari and Firefox on the same computer, with a copy-pasted bookmark link. It does not occur on a html page in Chrome. Chrome appears to be up to date, and attempts at using other the few other available pdf viewers in chrome are not successful. Javascript is enabled, as are ads and popups. This occurs with all usual extensions and without all extensions using incognito mode.  

<img width="1278" alt="screen shot 2018-05-20 at 1 17 44 pm" src="https://user-images.githubusercontent.com/8173741/40281524-25efbc0c-5c31-11e8-84dd-964692266bb5.png">
<img width="1280" alt="screen shot 2018-05-20 at 1 17 33 pm" src="https://user-images.githubusercontent.com/8173741/40281522-22880d1c-5c31-11e8-959f-963a8a3da0f0.png">

<img width="644" alt="screen shot 2018-05-20 at 1 16 21 pm" src="https://user-images.githubusercontent.com/8173741/40281532-3013b7d8-5c31-11e8-9b22-21806924be52.png">
<img width="1279" alt="screen shot 2018-05-20 at 1 17 54 pm" src="https://user-images.githubusercontent.com/8173741/40281527-29ba8df8-5c31-11e8-8a32-7f72bcb5ca45.png">

<img width="1275" alt="screen shot 2018-05-20 at 1 29 15 pm" src="https://user-images.githubusercontent.com/8173741/40281588-d949cacc-5c31-11e8-9c20-18e3be73d180.png">

**Comment by gkiar** [comment]
Hey @nirtiac, thanks for the note! Could you please try opening the developer console (on Mac+Chrome, this is `cmd+opt+i`) when viewing a pdf, then click the bookmarklet and see what shows up in the `console` tab? That would help diagnose what's happening. Thanks!

**Comment by nirtiac** [comment]
Hey, I get the message below.... but I get it on every page including simple html where the bookmarklet works. 

<img width="1279" alt="screen shot 2018-05-22 at 3 08 53 pm" src="https://user-images.githubusercontent.com/8173741/40384735-6fad4b6e-5dd2-11e8-87c4-c1eadadd2a47.png">
<img width="1278" alt="screen shot 2018-05-22 at 3 13 30 pm" src="https://user-images.githubusercontent.com/8173741/40384824-b4f72690-5dd2-11e8-8841-721f89034eb5.png">

```
embedded.html?name=&url=http://metalearning.ml/papers/metalearn17_furlanello.pdf&token=***&repo=nirtiac/reading&button=true:89

 Uncaught TypeError: Cannot read property 'url' of undefined
    at XMLHttpRequest.updateList (embedded.html?name=&url=http://metalearning.ml/papers/metalearn17_furlanello.pdf&token=***&repo=nirtiac/reading&button=true:89)
updateList @ embedded.html?name=&url=http://metalearning.ml/papers/metalearn17_furlanello.pdf&token=***&repo=nirtiac/reading&button=true:89
XMLHttpRequest.send (async)
(anonymous) @ embedded.html?name=&url=http://metalearning.ml/papers/metalearn17_furlanello.pdf&token=***&repo=nirtiac/reading&button=true:83
(anonymous) @ embedded.html?name=&url=http://metalearning.ml/papers/metalearn17_furlanello.pdf&token=***&repo=nirtiac/reading&button=true:116
load (async)
(anonymous) @ embedded.html?name=&url=http://metalearning.ml/papers/metalearn17_furlanello.pdf&token=***&repo=nirtiac/reading&button=true:40

```

I poked around the developer panel and have yet to see any evidence as to why the pdf does not load. The source looks fine.

<img width="1280" alt="screen shot 2018-05-22 at 3 10 26 pm" src="https://user-images.githubusercontent.com/8173741/40384754-7ab889ec-5dd2-11e8-9cf8-71481babc15a.png">
<img width="1280" alt="screen shot 2018-05-22 at 3 10 49 pm" src="https://user-images.githubusercontent.com/8173741/40384755-7ae0260a-5dd2-11e8-90f0-3aa7319380ad.png">

**Comment by gkiar** [comment]
Hey - Thanks for the update and sorry for the delay. I finally updated my chrome and notice that now PDFs (not webpages) disappear, as well. There is no error in the console either, so, sadly, I believe this is just the way Chrome wants to handle PDFs now - not as webpages, but as static files it downloads then views, meaning we won't be able to run javascript over it.

To test this, I also tried running Mendeley's bookmarklet over a raw PDF, and it wasn't able to load.

Sorry about this - paper landing pages/html versions are where it looks like we'll have to document these from 😞
